### ISSUE TITLE
App Hangs - OS X El Capitan

### ISSUE THREAD

**Issue by swainjo** [gephi/gephi#1241]
Steps to replicate:

Open attached .graphml file

- When Overview Loads, click Preview
- Click Overview to return to Overview module
- App hangs

System Info:

System Software Overview:

  System Version:	OS X 10.11.1 (15B42)
  Kernel Version:	Darwin 15.0.0

Hardware Overview:

  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro11,3
  Processor Name:	Intel Core i7
  Processor Speed:	2.3 GHz
  Number of Processors:	1
  Total Number of Cores:	4
  L2 Cache (per Core):	256 KB
  L3 Cache:	6 MB
  Memory:	16 GB
  Boot ROM Version:	MBP112.0138.B16
  SMC Version (system):	2.19f12
  Serial Number (system):	C02LH0WKFD57
  Hardware UUID:	F0E11056-0AC0-568C-A989-E69C866FFF04

[MeatFreeDec16Dec16_94weak.graphml.zip](https://github.com/gephi/gephi/files/68390/MeatFreeDec16Dec16_94weak.graphml.zip)

**Comment by mbastian** [comment]
Thanks for the report. I unfortunately can't reproduce the issue although I have a very similar configuration. Can you copy/paste the content of the log file? There might be useful information. You can obtain it at ~/Library/Application\ Support/gephi/0.9.0/dev/var/log/messages.log

**Comment by swainjo** [comment]
Here is the log file. I deleted the existing ones before running so this is just the exact steps in the ticket. 

[messages.log.zip](https://github.com/gephi/gephi/files/68406/messages.log.zip)

**Comment by mbastian** [comment]
Thanks, unfortunately nothing really informative in there. Does the app crash? When it hangs, can you still click on anything? Access the menubar? Does it hang before the Overview has loaded? Or after?

**Comment by swainjo** [comment]
I hangs before loading the Overview module.

The window becomes slightly smaller then just hangs with spinning beachball. Can't click anything and have to Force Quit.

**Comment by zebraengine** [comment]
I'm experiencing the same issue, Gephi 0.9.0 hanging when trying to switch from Data Laboratory to Overview. I'm running Gephi on Mac OS X 10.9.5 with Java SE Runtime Environment (build 1.8.0_31-b13). The issue appears regardless if i have a graph file loaded or not, so it doesn't appear to be something unique about the file I'm trying to work with.

**Comment by mbastian** [comment]
Ok, @zebraengine can you also share you messages.log? There's the GPU reference in there and I want to see if there's a pattern there. Thanks.

**Comment by dkbohl** [comment]
I am also experiencing the same. Can't find one particular magic combination that will replicate it, but it generally happens a minute or so into working with any graph. I've attached a few logs. Please let me know if there is anything else I can do to help.
[messages.zip](https://github.com/gephi/gephi/files/68576/messages.zip)

**Comment by zebraengine** [comment]
Hi @mbastian  - i may be missing something - but i'm not getting messages.log files when i force quite the application. Is there a specific step i should take in order to produce that file?

**Comment by bernorieder** [comment]
Same here on OS X 10.11.2 and Java 1.8.0_65-b17. Hang produces a wheel of death. I have noticed that the graph viewport is initialized incorrectly (too low in the panel, see https://www.dropbox.com/s/bv5tmdvko9ftz3c/gephi_hanging.png), maybe that's a direction to investigate.

**Comment by hpiedcoq** [comment]
Same for me.

**Comment by mbastian** [comment]
Ok, it probably is related to JOGL (the OpenGL library we're using) and what happens at the creation/destruction of the window. If someone if volunteer to help me debug this, let me know. We would run gephi from sources and try a few things. For example, go back to the previous version of JOGL (2.3.1) is something to try as I suspect an issue on their end.

**Comment by bernorieder** [comment]
@mbastian I can help, but I suck at Java; should be enough to run some sources, though

**Comment by swainjo** [comment]
I can probably help out tomorrow if you still need some debug code running.

**Comment by mbastian** [comment]
Cool, let's try this first: I created a build (Mac OS X) based on the JOGL version 2.3.1 instead of 2.3.2. Can you please try it and let me know if you have the same issue? Thanks https://www.dropbox.com/s/heszzmncwv8jvx0/gephi-0.9.0_dev_issue1241.dmg?dl=0

**Comment by dkbohl** [comment]
Hey @mbastian, unfortunately I'm still running into the same hang with the build you just posted.

**Comment by richgor** [comment]
**dkbohl** & **swainjo** & **zebraengine** & **dkbohl** & **hpiedcoq** -- have you installed the newest updates (as prompted in the lower right of the Gephi screen)?  Gephi 0.9 is working fine for me -- as long as I don't install those updates.  On one of my Macs, I installed the newest updates, and I have had the "hang" problem on that Mac. On two other Macs where the updates were not installed, everything is fine.  Unfortunately, I can't figure out a way to revert to the pre-updated version (even by reinstalling the original download file).  I've been documenting my experience here: https://github.com/gephi/gephi/issues/1253

**Comment by yannis1962** [comment]
I have the same problem on El Capitan 10.11.2. Everything opens correctly but I can't focus on the application and the colored wheel pointer keeps spinning. Any solution yet?

**Comment by bgbg** [comment]
This is how I solve this problem:
1. Remove any Gephi installations
2. Install Gephi
3. Open Gephi -- everything works
4. Close Gephi
5. Open Gephi again -- doesn't work
6. Kill Gephi
7. GOTO 1

Yes, removing Gephi installation and installing it once more, solves the problem until the next time you exit Gephi

**Comment by mbastian** [comment]
Seriously? That doesn't make any logical sense. This issue is getting more and more random o_O I hope to get to the bottom of it soon.

**Comment by bgbg** [comment]
tested this four times

**Comment by richgor** [comment]
For what it's worth, yesterday I was able to solve a "hang" problem with a somewhat simpler workaround ... namely, just move the Library/Application Support/gephi/0.9.0 folder to the trash and then relaunch Gephi.  No re-installation needed.

**Comment by kikohs** [comment]
Same issue here. On El Capitan 10.11.2

**Comment by koalaGreener** [comment]
Same problem

**Comment by fblamanna** [comment]
Same problem, Mac OS 10.11.2. It is really annoying.

**Comment by fblamanna** [comment]
@richgor your workaroung works also for me thanks!

**Comment by mbastian** [comment]
I want to further investigate where exactly it hangs and if it's consistent between users/execution. I made a version that logs more things related to the visualization window. Can you please try this version (https://dl.dropboxusercontent.com/u/481632/gephi-0.9.0_dev_issue_1241.dmg) and then attach the `messages.log` file (~/Library/Application\ Support/gephi/0.9.0/dev/var/log/messages.log) so I can see what's going on. I would also be interested if you see interesting messages in the OS X Console app (Other > Console) related to this hang. Thanks!

**Comment by richgor** [comment]
Thanks, @mbastian.  As a practical matter, now that my class has started, I may not be in a position to try a different build for a while.  Hoping that others who've been having the problem will try it out and get info to you.  cc: @dkbohl @swainjo @zebraengine @dkbohl @hpiedcoq @fblamanna @koalaG @kikohs @bgbg

**Comment by dkbohl** [comment]
Here is my first dump. Thanks so much!
[MessagesAndLogs.zip](https://github.com/gephi/gephi/files/81541/MessagesAndLogs.zip)

**Comment by mbastian** [comment]
@dkbohl Thanks! Can you clarify the steps you took? Did it hang during switch between datalab/overview or at start? Also, it seems that you opened a project and that actually threw an exception, which might be a different problem. Can you try without opening a project or use the default LesMiserables sample? Thanks.

**Comment by dkbohl** [comment]
Sorry about that, those last files were from me opening up an older project. I have now attached logs that are from a sessions that (1) opened up LesMis and accepted the default import settings, (2) switched to DataLab, (3) switched back to Overview upon which Gephi began to hang, and (4) force quit. Hope its helpful.
[Messages_Console.zip](https://github.com/gephi/gephi/files/81650/Messages_Console.zip)

**Comment by mbastian** [comment]
Thanks, that helps! Can you please try if the following scenario also hangs:
1. Open Gephi
2. Click on the X cross to close the Graph window/tab
3. Go to the menubar and select Window > Graph

**Comment by mbastian** [comment]
Just trying things, please let me know if this version hangs as well: https://dl.dropboxusercontent.com/u/481632/gephi-0.9.0_dev_issue_1241_2.dmg. By the way, if a nice person could arrange a remote control of a Mac that hangs and let me play with it this would likely go much faster :)

**Comment by dkbohl** [comment]
Hey @mbastian, I haven't been able to try out the scenario or new build, but I'd be happy to set up a time for you to remote in.

**Comment by dkbohl** [comment]
Okay, just tried the scenario. Closings the Graph window and opening it again from the menubar did not cause Gephi to crash, but switching to the Lab and back did. Here is the log using the build you posted yesterday.
[messages.log.zip](https://github.com/gephi/gephi/files/83501/messages.log.zip)

**Comment by mbastian** [comment]
Ok, that's interesting. Can anyone else on this thread try this? Would like to confirm it's consistent. Thanks.

**Comment by yannis1962** [comment]
dear Bastian,

I downloaded and launched version 1241_2 of Gephi. The behavior is exactly the same: application opens, all windows are there but the pointer keeps spinning and I have no control on anything. Here are the corresponding lines from the Console:

Jan  9 11:56:26 phoebe-1 kernel[0]: hfs: mounted Gephi 0.9.0 on device disk5s1
Jan  9 11:56:26 phoebe-1 mds[69]: (Volume.Normal:2464) volume:0x7fc58e863800 ********** Bootstrapped Creating a default store:1 SpotLoc:(null) SpotVerLoc:(null) occlude:0 /Volumes/Gephi 0.9.0
Jan  9 11:56:39 phoebe-1 Default Folder X[1793]: AppLaunch gephi : java (6893)
Jan  9 11:56:43 phoebe-1 Default Folder X[1793]: WindowCreated gephi : waiting for 0x7ff928324950
Jan  9 11:57:39 phoebe-1 bird[373]: LaunchServices: Failed to create bundleProxy for bundle org.gephi
 
I attach the messages.log file.

Thanks in advance for your help

Yannis




> Le 7 janv. 2016 à 22:29, Mathieu Bastian <notifications@github.com> a écrit :
> 
> Just trying things, please let me know if this version hangs as well: https://dl.dropboxusercontent.com/u/481632/gephi-0.9.0_dev_issue_1241_2.dmg <https://dl.dropboxusercontent.com/u/481632/gephi-0.9.0_dev_issue_1241_2.dmg>. By the way, if a nice person could arrange a remote control of a Mac that hangs and let me play with it this would likely go much faster :)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/gephi/gephi/issues/1241#issuecomment-169810124>.
> 

-- 
-------------------------------------------------------
Yannis Haralambous
Professor
Institut Mines-Télécom, Télécom Bretagne
Computer Science Department
UMR CNRS 6285 Lab-STICC
Technopôle Brest Iroise
CS 83818, 29238 Brest Cedex 3, France
Email: yannis.haralambous@telecom-bretagne.eu <mailto:yannis.haralambous@telecom-bretagne.eu>
Internet: http://perso.telecom-bretagne.eu/yannisharalambous/ <http://omega.enstb.org/yannis>
ICBM address: 48°21'31.57"N 4°34'16.76"W
Twitter: y_haralambous
-------------------------------------------------------
...the ball I threw while playing in the park
has not yet reached the ground
(Dylan Thomas)

Es gab eine Zeit, wo ich nur ungern über Schubert sprechen,
nur Nächtens den Bäumen und Sternen von ihm vorerzählen mögen.
(Robert Schumann)

**Comment by swainjo** [comment]
Ok, I get the same error - here is the log file.
[messages.log.zip](https://github.com/gephi/gephi/files/85867/messages.log.zip)

**Comment by carlosguadian** [comment]
Same problem here. Gephi freeze when I try to change from Data Laboratory to Overview.
[messages.log.zip](https://github.com/gephi/gephi/files/89178/messages.log.zip)

**Comment by carlosguadian** [comment]
If I reinstall the app works fine, but I get a warning with the following message when I change from Overview to Data Laboratory:

java.lang.NullPointerException
	at org.gephi.desktop.datalab.DataTableTopComponent$2.run(DataTableTopComponent.java:372)
	at java.util.TimerThread.mainLoop(Timer.java:555)
[catch] at java.util.TimerThread.run(Timer.java:505)

I hope it will be useful!

**Comment by mbastian** [comment]
@dkbohl Appreciate the help with the remote. Can you send me a quick note at mathieu.bastian [at] gephi.org so we can schedule something together. I would like to try the free Chrome Remote Desktop extension https://chrome.google.com/webstore/detail/chrome-remote-desktop/gbchcmhmhahfdphkhkmpfmihenigjmpp/related?hl=en

**Comment by ChrisHH** [comment]
I installed the latest version of gephi in december and had no problems. Until today, when this issue came up - and I did not change anything on my mac. As far as I can see, there was no Apple update as well.

**Comment by ChrisHH** [comment]
@mbastian 

I checked that now:

https://dl.dropboxusercontent.com/u/481632/gephi-0.9.0_dev_issue_1241_2.dmg

And it works; the issue is gone.

**Comment by dkbohl** [comment]
Now I have only been able to consistently reproduce this hang when Gephi is fully maximized. If after opening Gephi, I resize the window so that it does not take up the full screen, I can switch between Data Lab, Preview, and Overview without problem (however often times Gephi will automatically resize to the full screen after switching views). If Gephi is maximized I encounter hang when switching from (1) DataLab to Overview and from (2) Preview to Overview, but not from (3) DataLab to Preview or from (4) Preview to DataLab.

[messages.zip](https://github.com/gephi/gephi/files/94695/messages.zip)
[Console-Gephi.log.zip](https://github.com/gephi/gephi/files/94700/Console-Gephi.log.zip)

**Comment by swainjo** [comment]
@dkbohl - I can also replicate similar behaviour. If I resize the windows (I have to do this separately for each view) then it happily switches between views. If I only resize one of the views then Gephi will resize the windows.

**Comment by mbastian** [comment]
There's something going on with the underlying Window System that isn't right and makes the app hang.  I was finally able to reproduce this hang when I lowered the resolution of my Macbook (retina) from 1440x900 to 1280x800 but I'm still unsure about the root cause. It seems to be related to the content of the `~/Library/Application Support/gephi/0.9.0dev/config/Windows2Local-overview/WindowManager.wswmgr` file, and specifically the width/height in the joined-properties attribute. Maybe if that's larger than the actual screen size it hangs. If I exclude the scenario where I reduce the resolution size manually I'm not sure why this value would be higher than the screen size. Can everyone who still can reproduce this issue copy/paste the content of this file here? Thanks!

**Comment by dkbohl** [comment]
I'm running a 15.4-inch retina at 2880 x 1800. Here is the contents of WindowManager.wswmgr copied after I forced quit from a hang:

<?xml version="1.0" encoding="UTF-8"?>

<windowmanager version="2.1">
    <main-window>
  <joined-properties
   x="0"
   y="23"
   width="1440"
   height="437"
   relative-x="0.0"
   relative-y="0.0"
   relative-width="0.0"
   relative-height="0.0"
   centered-horizontally="false"
   centered-vertically="false"
   maximize-if-width-below="0"
   maximize-if-height-below="0"
   frame-state="6"
/>
  <separated-properties
   x="144"
   y="87"
   width="1152"
   height="69"
   relative-x="0.0"
   relative-y="0.0"
   relative-width="0.0"
   relative-height="0.0"
   centered-horizontally="false"
   centered-vertically="false"
   frame-state="0"
/>
  </main-window>
    <editor-area state="joined" frame-state="0">
  <constraints>
  <path orientation="horizontal" number="60" weight="0.7" />
  <path orientation="vertical" number="40" weight="0.7" />
  <path orientation="horizontal" number="40" weight="0.9091890724543786" />
  <path orientation="vertical" number="40" weight="0.5" />
  </constraints>
  <bounds x="475" y="209" width="604" height="384" />
    </editor-area>
    <screen width="1440" height="900"/>
    <active-mode name="editor"/>
    <toolbar configuration="Standard" preferred-icon-size="24"/>
    <tc-list>
  <tc-id id="AppearanceTopComponent"/>
  <tc-id id="StatisticsTopComponent"/>
  <tc-id id="LayoutTopComponent"/>
  <tc-id id="FiltersTopComponent"/>
  <tc-id id="MultiModeWindowTopComponent"/>
  <tc-id id="ContextTopComponent"/>
    </tc-list>
</windowmanager>

**Comment by mbastian** [comment]
I may have identified the root cause and can't reproduce the issue anymore after a fix. Please try the following version as well: https://www.dropbox.com/s/bcja4opm5nqibwl/gephi-0.9.0_dev_issue1241_3.dmg?dl=0 But make sure to delete your user folder at `~/Library/Application Support/gephi/0.9.0` beforehand otherwise the issue will remain.

**Comment by carlosguadian** [comment]
I've try and at the moment works ok, I can't reproduce the problem. Greetings!

**Comment by yannis1962** [comment]
I did as you say (removing the folder, then installing the new version) and I have exactly the same problem: window opens and colored wheel cursor spins indefinitely. Is there anything I can send you?

> Le 20 janv. 2016 à 17:45, Mathieu Bastian <notifications@github.com> a écrit :
> 
> I may have identified the root cause and can't reproduce the issue anymore after a fix. Please try the following version as well: https://www.dropbox.com/s/bcja4opm5nqibwl/gephi-0.9.0_dev_issue1241_3.dmg?dl=0 <https://www.dropbox.com/s/bcja4opm5nqibwl/gephi-0.9.0_dev_issue1241_3.dmg?dl=0> But make sure to delete your user folder at ~/Library/Application Support/gephi/0.9.0 beforehand otherwise the issue will remain.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/gephi/gephi/issues/1241#issuecomment-173267899>.
> 

-- 
-------------------------------------------------------
Yannis Haralambous
Professor
Institut Mines-Télécom, Télécom Bretagne
Computer Science Department
UMR CNRS 6285 Lab-STICC
Technopôle Brest Iroise
CS 83818, 29238 Brest Cedex 3, France
Email: yannis.haralambous@telecom-bretagne.eu <mailto:yannis.haralambous@telecom-bretagne.eu>
Internet: http://perso.telecom-bretagne.eu/yannisharalambous/ <http://omega.enstb.org/yannis>
ICBM address: 48°21'31.57"N 4°34'16.76"W
Twitter: y_haralambous
-------------------------------------------------------
...the ball I threw while playing in the park
has not yet reached the ground
(Dylan Thomas)

Es gab eine Zeit, wo ich nur ungern über Schubert sprechen,
nur Nächtens den Bäumen und Sternen von ihm vorerzählen mögen.
(Robert Schumann)

**Comment by dkbohl** [comment]
Also still experiencing the hang. Here are the messages and relevant console logs:
[Archive.zip](https://github.com/gephi/gephi/files/97839/Archive.zip)

**Comment by bill10** [comment]
Same problem for me. I removed the user folder and then installed  https://www.dropbox.com/s/bcja4opm5nqibwl/gephi-0.9.0_dev_issue1241_3.dmg?dl=0. Still hangs

**Comment by mbastian** [comment]
@dkbohl, @bill10 and @yannis1962 would you mind archiving and attaching your `~/Library/Application\ Support/gephi/0.9.0/dev/config/` folder? I want to investigate various files in there. Thanks!

**Comment by yannis1962** [comment]
Is that OK?

> Le 27 janv. 2016 à 11:45, Mathieu Bastian <notifications@github.com> a écrit :
> 
> @dkbohl <https://github.com/dkbohl>, @bill10 <https://github.com/bill10> and @yannis1962 <https://github.com/yannis1962> would you mind archiving and attaching your ~/Library/Application\ Support/gephi/0.9.0/dev/config/ folder? I want to investigate various files in there. Thanks!
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/gephi/gephi/issues/1241#issuecomment-175547539>.
> 

-- 
-------------------------------------------------------
Yannis Haralambous
Professor
Institut Mines-Télécom, Télécom Bretagne
Computer Science Department
UMR CNRS 6285 Lab-STICC
Technopôle Brest Iroise
CS 83818, 29238 Brest Cedex 3, France
Email: yannis.haralambous@telecom-bretagne.eu <mailto:yannis.haralambous@telecom-bretagne.eu>
Internet: http://perso.telecom-bretagne.eu/yannisharalambous/ <http://omega.enstb.org/yannis>
ICBM address: 48°21'31.57"N 4°34'16.76"W
Twitter: y_haralambous
-------------------------------------------------------
...the ball I threw while playing in the park
has not yet reached the ground
(Dylan Thomas)

Es gab eine Zeit, wo ich nur ungern über Schubert sprechen,
nur Nächtens den Bäumen und Sternen von ihm vorerzählen mögen.
(Robert Schumann)

**Comment by dkbohl** [comment]
Here you go.
[config.zip](https://github.com/gephi/gephi/files/106732/config.zip)

**Comment by bill10** [comment]
No problem.  
[config.zip](https://github.com/gephi/gephi/files/106835/config.zip)
Thank you!

**Comment by mbastian** [comment]
Thanks @dkbohl, @bill10 and @yannis1962. I'm finally able to reproduce this issue (or at least I'm able to reproduce its consequences) and track it down to a deadlock happening in JOGL, the OpenGL library we're using. I've used the JConsole tool to look at where the threads are stuck and I see that the EDT and the NEWT threads are waiting on each other:

```Name: AWT-EventQueue-0
State: WAITING on java.lang.Object@2dbec237 owned by: AWT-EventQueue-0-Display-.macosx_nil-1-EDT-1
Total blocked: 70  Total waited: 40

Stack trace: 
java.lang.Object.wait(Native Method)
java.lang.Object.wait(Object.java:502)
jogamp.newt.DefaultEDTUtil.invokeImpl(DefaultEDTUtil.java:242)
jogamp.newt.DefaultEDTUtil.invoke(DefaultEDTUtil.java:165)
jogamp.newt.DisplayImpl.runOnEDTIfAvail(DisplayImpl.java:442)
jogamp.newt.WindowImpl.runOnEDTIfAvail(WindowImpl.java:2782)
jogamp.newt.WindowImpl.setVisible(WindowImpl.java:1330)
jogamp.newt.WindowImpl.setVisible(WindowImpl.java:1335)
com.jogamp.newt.opengl.GLWindow.setVisible(GLWindow.java:578)
com.jogamp.newt.awt.NewtCanvasAWT.attachNewtChild(NewtCanvasAWT.java:1057)
com.jogamp.newt.awt.NewtCanvasAWT.validateComponent(NewtCanvasAWT.java:956)
com.jogamp.newt.awt.NewtCanvasAWT.reshape(NewtCanvasAWT.java:750)
   - locked java.lang.Object@67faa7e9
   - locked java.awt.Component$AWTTreeLock@4b9aeb19
java.awt.Component.setBounds(Component.java:2263)
...
```

And the AWT-EventQueue-0-Display-.macosx_nil-1-EDT-1 thread has the following stack: 

```Name: AWT-EventQueue-0-Display-.macosx_nil-1-EDT-1
State: WAITING on java.lang.Object@f8a24ae
Total blocked: 3  Total waited: 4

Stack trace: 
java.lang.Object.wait(Native Method)
java.lang.Object.wait(Object.java:502)
jogamp.nativewindow.macosx.OSXUtil.RunOnMainThread(OSXUtil.java:278)
jogamp.newt.driver.macosx.WindowDriver.createWindow(WindowDriver.java:777)
jogamp.newt.driver.macosx.WindowDriver.reconfigureWindowImpl(WindowDriver.java:445)
jogamp.newt.driver.macosx.WindowDriver.createNativeImpl(WindowDriver.java:198)
jogamp.newt.WindowImpl.createNative(WindowImpl.java:751)
jogamp.newt.WindowImpl.setVisibleActionImpl(WindowImpl.java:1248)
jogamp.newt.WindowImpl$VisibleAction.run(WindowImpl.java:1318)
com.jogamp.common.util.RunnableTask.run(RunnableTask.java:145)
   - locked java.lang.Object@2dbec237
jogamp.newt.DefaultEDTUtil$NEDT.run(DefaultEDTUtil.java:375)```

Since then I've tried to understand why this is happening and tried a few different things controlling how things are initialized. That didn't work so I've decided to switch back using `GLCanvas` instead of `NewtCanvasAWT` on Mac OS. The `Newt` canvas has some advantages but nothing that really can't be done with the `GLCanvas`. The Newt scheme specifically tries to do things in a separate thread (not the EDT) and therefore is exposed to problems of this kind. As expected the problem doesn't appear when using GLCanvas instead so that may be the road going forward. I've built a version with these changes here: https://www.dropbox.com/s/kf6dmvkjsl5qlre/gephi-0.9.0_dev_issue1241_4.dmg?dl=0 
Can you try and let me know if the problem still exist? Thanks!

**Comment by dkbohl** [comment]
@mbastian, that seems to have done it for me! Thank you so much!

**Comment by bill10** [comment]
@mbastian that nails the problem! thank you!

**Comment by mbastian** [comment]
Nice, happy to hear. Hopefully that fixes it for everyone! A 0.9.1 release will be done soon so this patch and all the others can be included.

**Comment by yannis1962** [comment]
halleluyah !!!
thank you so much

> Le 1 févr. 2016 à 19:14, Mathieu Bastian <notifications@github.com> a écrit :
> 
> Nice, happy to hear. Hopefully that fixes it for everyone! A 0.9.1 release will be done soon so this patch and all the others can be included.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/gephi/gephi/issues/1241#issuecomment-178106933>.
> 

-- 
-------------------------------------------------------
Yannis Haralambous
Professor
Institut Mines-Télécom, Télécom Bretagne
Computer Science Department
UMR CNRS 6285 Lab-STICC
Technopôle Brest Iroise
CS 83818, 29238 Brest Cedex 3, France
Email: yannis.haralambous@telecom-bretagne.eu <mailto:yannis.haralambous@telecom-bretagne.eu>
Internet: http://perso.telecom-bretagne.eu/yannisharalambous/ <http://omega.enstb.org/yannis>
ICBM address: 48°21'31.57"N 4°34'16.76"W
Twitter: y_haralambous
-------------------------------------------------------
...the ball I threw while playing in the park
has not yet reached the ground
(Dylan Thomas)

Es gab eine Zeit, wo ich nur ungern über Schubert sprechen,
nur Nächtens den Bäumen und Sternen von ihm vorerzählen mögen.
(Robert Schumann)

**Comment by mbastian** [comment]
The fix has been deployed to updates. If you do Help > Check for updates and then restart Gephi it will work. As said, it will be included in the upcoming 0.9.1 release as well.

**Comment by hpiedcoq** [comment]
Congrats for this updated version! I had the same issue on my mac.
It works really smoothly now.
